hack/utilities.sh

```diff
@@ -75,3 +75,12 @@ function retry() {
     ((N++))
   done
 }
+
+# Some useful colors.
+if [[ -z "${color_start-}" ]]; then
+  declare -r color_start="\033["
```

Comment:
Yes. I think in order to defer to the style guide as an authoritative guide that governs our scripts, we'll need to agree to it as a community.

My intention was not to defer to it (i.e. use the guide as a justification to demand a change). I merely pointed out its existence, observed I'd found it useful and even highlighted that it may not have any bearing (anticipating need for agreement before it can be used in any meaningful way).